README.md

```diff
@@ -141,7 +141,7 @@ Add to your `.babelrc`'s plugins array.
 ## Benchmarks
 > Bootstrap: `npm run bootstrap`
 
-> Running the benchmarks: `./scripts/benchmark.js file.js`
+> Running the benchmarks: `./scripts/benchmark.js package` for the main file **or** `./scripts/benchmark.js package path/file.js` for a specific file in the package
```

Comment:
Might be worth going with an optional argument doc style e.g. -
`./scripts/benchmark.js package [relative path]`
The last argument is optional and defaults to the package's main file.